src/Common/Bus/Integration/Bundle/GamingPlatformBusBundle.php

```diff
@@ -0,0 +1,140 @@
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
+
+use function Symfony\Component\DependencyInjection\Loader\Configurator\abstract_arg;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\tagged_locator;
+
+final class GamingPlatformBusBundle extends AbstractBundle implements CompilerPassInterface
+{
+    /**
+     * @var array{buses: array<int, array{serviceId: string, handlerTagName: string}>}
+     */
+    private array $passConfig = ['buses' => []];
+
+    public function build(ContainerBuilder $container): void
+    {
+        $container->addCompilerPass($this);
+    }
+
+    public function configure(DefinitionConfigurator $definition): void
+    {
+        $definition->rootNode()
+            ->children()
+                ->arrayNode('buses')
+                ->defaultValue(['default' => null])
+                ->requiresAtLeastOneElement()
+                ->prototype('variable')
+                ->end()
+            ->end();
+    }
+
+    /**
+     * @param array{buses: array<string, ?array{serviceId?: string, handlerTagName?: string}>} $config
+     */
+    public function loadExtension(array $config, ContainerConfigurator $container, ContainerBuilder $builder): void
+    {
+        $extensionName = $this->getContainerExtension()?->getAlias();
+
+        foreach ($config['buses'] as $name => $busConfig) {
+            $serviceId = $busConfig['serviceId'] ?? $extensionName . '.' . $name;
+            $handlerTagName = $busConfig['handlerTagName'] ?? $extensionName . '.' . $name . '.handler';
+
+            $container->services()
+                ->set($serviceId, RoutingBus::class)
+                ->arg(0, tagged_locator($handlerTagName))
+                ->arg(1, abstract_arg('routes'));
+
+            $this->passConfig['buses'][] = ['serviceId' => $serviceId, 'handlerTagName' => $handlerTagName];
+        }
+    }
+
+    public function process(ContainerBuilder $container): void
+    {
+        foreach ($this->passConfig['buses'] as $bus) {
+            $this->processBus($container, $bus['serviceId'], $bus['handlerTagName']);
+        }
+    }
+
+    private function processBus(ContainerBuilder $container, string $serviceId, string $handlerTagName): void
+    {
+        $routes = [];
+
+        foreach ($container->findTaggedServiceIds($handlerTagName) as $taggedServiceId => $tagConfig) {
+            $taggedServiceClass = $container->getDefinition($taggedServiceId)->getClass() ?? throw new Exception(
+                'No class defined for service "' . $taggedServiceId . '".'
+            );
+            $addRoute = static function (string $type, string $method) use ($taggedServiceId, &$routes) {
+                $routes[$type] = ['serviceId' => $taggedServiceId, 'method' => $method];
```

Comment:
To avoid confusion to which destination the request is sent at the end (currently, the last one wins), this should throw an exception if a route for a `$type` already exists. Should be pretty rare, but still worth to handle explicitly. Additionally, there should also be an option to ignore routes, e.g:
```
gaming_platform_bus:
  buses:
    identity_command:
      ignoredRoutes:
        - { type: 'FCQN\To\Request\Type' serviceId: some.service.id, method: someMethod }
        - { type: 'FCQN\To\Another\Request\Type' serviceId: some.service.id, method: someMethod }
```

Reply:
It's checked, but there's no added complexity with `ignoredRoutes`. In that rare cases, it might be better to just manually route the request types.